tests/test_django_orm.py

```diff
@@ -61,7 +61,7 @@ def test_field_unpickled(self):
     def test_field_pickled(self):
         prep_value = self.field.get_db_prep_value(self.credentials,
                                                   connection=None)
-        self.assertEqual(prep_value, self.pickle)
+        self.assertEqual(_to_bytes(prep_value), self.pickle)
```

Comment:
It seems `get_db_prep_value` is still defined. Is this because it relies on the methods we have implemented?